docs/troubleshooting.md

```diff
@@ -3,51 +3,50 @@
 The Collector offers multiple ways to measure the health of the Collector
 as well as investigate issues.
 
+- The [zpages](https://github.com/open-telemetry/opentelemetry-collector/tree/master/extension/zpagesextension/README.md)
+  extension, which by default is on port `55679`, can be used to check
+  receivers and exporters trace operations via `/debug/tracez`. `zpages`
+  often contains error logs that the collector does not emit.
 - By default, Prometheus metrics are exposed on port `8888` and path
-`/metrics`. A grafana dashboard can be found
-[here](https://grafana.com/grafana/dashboards/11575).
-
+  `/metrics`. A grafana dashboard can be found
+  [here](https://grafana.com/grafana/dashboards/11575).
 - Logs can be helpful in identifying issues. The verbosity level, which
-defaults to `INFO` can also be adjusted by passing the `--log-level` flag
-to the `otelcol` process. See `--help` for more details.
-
-- [Local exporters](https://github.com/open-telemetry/opentelemetry-collector/tree/master/exporter#local-exporters)
-can be configured to inspect the data being processed by the Collector.
-
+  defaults to `INFO` can also be adjusted by passing the `--log-level` flag
+  to the `otelcol` process. See `--help` for more details.
```

Comment:
Even though my Windows Service PR hasn't been merged yet, I think it's worth adding a comment here that when run as a Windows Service, logs are redirected to Windows event log.